tests/tests_integration/test_api/test_time_series.py

```diff
@@ -38,7 +38,7 @@ def test_retrieve_multiple(self, cognite_client):
     def test_retrieve_multiple_ignore_unknown(self, cognite_client):
         res = cognite_client.time_series.list(limit=2)
         retrieved_assets = cognite_client.time_series.retrieve_multiple(
-            [t.id for t in res] + [0, 1, 2, 3], ignore_unknown_ids=True
+            [t.id for t in res] + [1, 2, 3], ignore_unknown_ids=True
```

Comment:
why this change?

Reply:
there was a change in the datapoints api, and now it checks that all received ids are positive numbers. That change broke this test.